Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py

```diff
@@ -858,3 +858,39 @@ def generate_login_url(client: MicrosoftClient) -> CommandResults:
 and paste it in your instance configuration under the **Authorization code** parameter.
     """
     return CommandResults(readable_output=result_msg)
+
+
+def get_from_args_or_params(args: Dict[str, Any], params: Dict[str, Any], key: str) -> Any:
+    """
+    Get a value from args or params.
+    this function is used in commands that have a value that can be provided in the instance parameters or in the command,
```

Comment:
Let's explain here as well that the arg overrides the param if both are provided.

Reply:
Fixed.